pkg/cmd/taskrun/list_test.go

```diff
@@ -143,6 +144,42 @@ func TestListTaskRuns(t *testing.T) {
 			args:     []string{"list", "-n", "random"},
 			expected: []string{emptyMsg, ""},
 		},
+		{
+			name:    "limit taskruns returned to 1",
+			command: command(t, trs, now),
+			args:    []string{"list", "-n", "foo", "-l", fmt.Sprintf("%d", 1)},
```

Comment:
How about replacing `fmt.Sprintf("%d", 1)` -> `"1"`?

Reply:
I tried using `"1"` previously in #185 and it did not work. The argument needs to be an int, so this is the way I found to pass it as a String and have it eventually interpreted as an int.

Reply:
:+1: